packages/airnode-node/src/constants.ts

```diff
@@ -35,3 +35,6 @@ export const MAXIMUM_ONCHAIN_ERROR_LENGTH = 100;
 
 // The maximum amount of time in milliseconds that pre and post processing is allowed to execute
 export const PROCESSING_TIMEOUT = 10_000;
+
+// The length of the hashed portion of a deployment ID
+export const DEPLOYMENT_ID_LENGTH = 8;
```

Comment:
There is already such a constant https://github.com/api3dao/airnode/blob/master/packages/airnode-node/src/workers/index.ts#L8. I'm fine with using either of these places but we should just use the one.

Reply:
I am using only one- I removed the one you link to, but it's probably cleaner to just export that. I'll update.